apps/server/src/modules/files/entity/file.entity.spec.ts

```diff
@@ -0,0 +1,309 @@
+import { ObjectId } from '@mikro-orm/mongodb';
+
+import { setupEntities, storageProviderFactory } from '@shared/testing';
+import { FileOwnerModel, FilePermissionReferenceModel } from '@src/modules/files/domain';
+
```

Comment:
Please remove new lines

Reply:
Let's continue the discussion on it under this comment https://github.com/hpi-schul-cloud/schulcloud-server/pull/4303#discussion_r1305536299 .